x/fungible/client/cli/tx_update_zrc20_paused_status.go

```diff
@@ -36,15 +36,45 @@ func CmdUpdateZRC20PausedStatus() *cobra.Command {
 				return cosmoserrors.Wrapf(sdkerrors.ErrInvalidRequest, "invalid action (%d)", action)
```

Comment:
There is no more action, can we remove it?

Reply:
fa956b75d21387138d7818aad1b02445dd1ac8e6